README.rst

```diff
@@ -23,3 +23,7 @@ InvenioRDM Research Data Management
 Further documentation is available on
 https://inveniordm.docs.cern.ch
 
+For Mirador Previewer
```

Comment:
why adding this here?
It would be better to add a section in the main doc of InvenioRDM, given more explanations about this feature.